include/bl_storage.h

```diff
@@ -238,6 +291,87 @@ NRFX_STATIC_INLINE void read_implementation_id_from_otp(uint8_t *buf)
  * This is a temporary solution until TF-M has access to NSIB functions.
  */
 
+#ifdef CONFIG_NRFX_RRAMC
```

Comment:
Since we are using "if defined", every other place, I suggest we do not mix it up.
```suggestion
#if defined(CONFIG_NRFX_RRAMC)
```